### DISCUSSION TITLE
DAI 2023-2024 - Users API - BetterBlood

### DISCUSSION THREAD

**Discussion by BetterBlood** [heig-vd-dai-course/heig-vd-dai-course#186] [Show and tell]
[repo](https://github.com/BetterBlood/21_curl.git)

**Comment by ludelafo** [comment]
Thank you!

What did you think of the new approach for the course material? The blending between the theoretical and practical content in one document?

**Reply by BetterBlood** [reply]
This way suit me well 👌🏼
